# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ppp/hotfix.patch.sparc
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ppp-ppp-2.4.9/pppd/Makefile.linux.vanilla	2022-04-22 14:51:27.403517525 +0200
+++ ppp-ppp-2.4.9/pppd/Makefile.linux	2022-04-22 14:51:34.633517898 +0200
@@ -68,7 +68,7 @@
 
 HAS_SHADOW=y
 #USE_PAM=y
-HAVE_INET6=y
+#HAVE_INET6=y
 
 # Enable plugins
 PLUGIN=y
